pandas/core/arrays/datetimes.py

```diff
@@ -231,8 +231,13 @@ def _simple_new(cls, values, freq=None, tz=None):
         result = object.__new__(cls)
         result._data = values
         result._freq = freq
-        tz = timezones.maybe_get_tz(tz)
-        result._tz = timezones.tz_standardize(tz)
+        if tz is None:
```

Comment:
is it cheap to detect when you need to call maybe_get_tz & tz_standarize? ideally these could be called in the DatetimeTZDtype constructor / alternatively could have a dedicate constructor, 

``DateteimeTZDtype.construct_from_tztype``, and will pave the way for a unified dtype as well.